src/renderers/noop/ReactNoop.js

```diff
@@ -160,6 +172,14 @@ var ReactNoop = {
     }
   },
 
+  hasScheduledDeferredCallback() {
+    return Boolean(scheduledDeferredCallback);
```

Comment:
I'd argue that maybe we shouldn't expose flushAnimationPri(), flushDeferredPri() neither.

It seems natural now but look at all the other tests we have to clean up now that seemed to expose natural implementation details in the past.